panda_moveit_config/launch/move_group.launch.py

```diff
@@ -268,7 +276,7 @@ def generate_launch_description():
         # move_servo
         Node(
             package="moveit_servo",
-            executable="servo_node_main",
+            executable="servo_node",
```

Comment:
```suggestion
            executable="servo_node_main",                     
```

Need to maintain the old name, otherwise not working.